ckks/decryptor.go

```diff
@@ -1,79 +1,26 @@
 package ckks
 
 import (
-	"github.com/ldsec/lattigo/v2/ring"
 	"github.com/ldsec/lattigo/v2/rlwe"
-	"github.com/ldsec/lattigo/v2/utils"
 )
 
-// Decryptor is an interface for decrypting Ciphertexts. A Decryptor stores the secret-key.
-type Decryptor interface {
-	// DecryptNew decrypts the ciphertext and returns a newly created
```

Comment:
It is probably better to keep an interface at the scheme's level too.

Reply:
fixed